pom.xml

```diff
@@ -28,7 +28,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <findbugs-maven-plugin.version>3.0.1</findbugs-maven-plugin.version>
+    <findbugs-maven-plugin.version>3.0.2</findbugs-maven-plugin.version>
```

Comment:
Never mix 2 unrelated changes in one when it possible. FB absolutely not related to `Added a new method to validate the GitHub API URL`